pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -216,7 +217,8 @@ func (l *listUsersQuery) expandDirect(
 		return err
 	}
 
-	iter, err := l.ds.Read(ctx, req.GetStoreId(), &openfgav1.TupleKey{
+	ds := storagewrappers.NewCombinedTupleReader(l.ds, req.GetContextualTuples().GetTupleKeys())
```

Comment:
suggestion: (non-blocking) Looks like there is a lot of duplicate code between expandDirect and expandTTU.  Anyway we can make some common function to avoid code duplication?